packages/blue-sdk-viem/src/signatures/index.ts

```diff
@@ -0,0 +1,14 @@
+import * as _Manager from "./manager";
+import * as _Permit from "./permit";
+import * as _Permit2 from "./permit2";
+import { verifySignature as _verifySignature } from "./utils";
+
+export * from "./types";
+
+export namespace SignatureUtils {
+  export const Permit = _Permit;
+  export const Permit2 = _Permit2;
+  export const Manager = _Manager;
+
+  export const verifySignature = _verifySignature;
+}
```

Comment:
Is this namespace really interesting? I feel like this is similar to ethers v5 design where you had to import blocks of code like `utils` and then you would use `utils.formatUnits` etc

I understand that here the dev has a choice to either use `SignatureUtils` or direct exports though. But it is really interesting to use a namespace

Reply:
I didn't think about this, I copied what was done in blue-sdk-ethers